src/dmd/cppmangle.d

```diff
@@ -64,6 +64,19 @@ const(char)* cppTypeInfoMangleItanium(Dsymbol s)
     return buf.extractString();
 }
 
+/******************************
+* Determine if a symbol is the 'primary' destructor, that is, the most-aggregate destructor (the one that is defined as __xdtor)
+*/
+bool isPrimaryDtor(Dsymbol sym)
```

Comment:
Ideally it should be `package` but I'll approve anyway.

Reply:
Yeah, make it `package` or document with all the appropriate `returns` and `params` DDoc sections.  And throw a newline
somewhere in the summary so it's not so long.